Sources/MapboxDirections/MBVisualInstructionType.swift

```diff
@@ -33,39 +33,11 @@ public enum VisualInstructionComponentType: Int, CustomStringConvertible {
     /**
      A component contains an exit number.
      */
-    case exitCode
+    case exitCode = "exit-number"
     
-    public init?(description: String) {
-        let type: VisualInstructionComponentType
-        switch description {
-        case "delimiter":
-            type = .delimiter
-        case "icon":
-            type = .image
-        case "text":
-            type = .text
-        case "exit":
-            type = .exit
-        case "exit-number":
-            type = .exitCode
-        default:
-            return nil
-        }
-        self.init(rawValue: type.rawValue)
-    }
+    /**
+    A component contains a lane.
+     */
+    case lane
```

Comment:
This case would never occur because the component would be deserialized as a LaneInstructionComponent instead of a VisualInstructionComponent.